packages/web-components/src/components/va-crisis-line-modal/va-crisis-line-modal.tsx

```diff
@@ -0,0 +1,130 @@
+import { Component, Host, State, h } from '@stencil/core';
+import arrowRightSvg from '../../assets/arrow-right-white.svg';
+import { CONTACTS } from '../../contacts';
+
+/**
+ * @componentName Crisis Line Modal
+ * @maturityCategory caution
+ * @maturityLevel candidate
+ */
+
+@Component({
+  tag: 'va-crisis-line-modal',
+  styleUrl: 'va-crisis-line-modal.scss',
+  shadow: true,
+})
+export class VACrisisLineModal {
+  @State() isOpen: boolean = false;
+
+  setVisible() {
+    this.isOpen = true;
+  }
+
+  setNotVisible() {
+    this.isOpen = false;
+  }
+
+  render() {
+    return (
+      <Host>
+        <div class="va-crisis-line-container">
+          <button
+            onClick={() => this.setVisible()}
+            data-show="#modal-crisisline"
+            class="va-crisis-line va-overlay-trigger"
+            part="button"
+          >
+            <div class="va-crisis-line-inner">
+              <span class="va-crisis-line-icon" aria-hidden="true"></span>
+              <span class="va-crisis-line-text">
+                Talk to the <strong>Veterans Crisis Line</strong> now
+              </span>
+              <img
+                class="va-crisis-line-arrow"
+                src={arrowRightSvg}
+                aria-hidden="true"
+              />
+            </div>
+          </button>
+        </div>
+        <va-modal
+          modalTitle="We’re here anytime, day or night – 24/7"
+          onPrimaryButtonClick={() => this.setNotVisible()}
+          onCloseEvent={() => this.setNotVisible()}
+          visible={this.isOpen}
+          large={true}
+        >
+          <div
+            id="modal-crisisline"
+            class="va-overlay va-modal va-modal-large"
+            role="alertdialog"
+          >
+            <div class="va-crisis-panel va-modal-inner">
+              <div class="va-overlay-body va-crisis-panel-body">
+                <p>
+                  If you are a Veteran in crisis or concerned about one, connect
+                  with our caring, qualified responders for confidential help.
+                  Many of them are Veterans themselves.
+                </p>
+                <ul class="va-crisis-panel-list">
+                  <li>
+                    <i
+                      class="fa fa-phone va-crisis-panel-icon"
+                      aria-hidden="true"
+                    />
+                    {
+                      <a href="tel:988">
+                        Call <strong>988 and select 1</strong>
+                      </a>
+                    }
+                  </li>
+                  <li>
+                    <i
+                      class="fa fa-mobile-alt va-crisis-panel-icon"
+                      aria-hidden="true"
+                    />
+                    <span>
+                      Text&nbsp;
+                      <va-telephone sms contact="838255" />
```

Comment:
The number portion of this link should be bold I think: 

![image](https://github.com/department-of-veterans-affairs/component-library/assets/15097156/bbcb38fe-c894-4231-b571-26ea6494f915)

Reply:
The number portion of that link is already bold.

Reply:
Sorry, I realize my screenshot probably wasn't very helpful haha. 

This is what I see on production: 
![image](https://github.com/department-of-veterans-affairs/component-library/assets/15097156/6363c560-837d-4c35-9e7d-8afaa0edf2cf)

And this is what I see on Chromatic:
![Screenshot from 2023-10-04 09-13-19](https://github.com/department-of-veterans-affairs/component-library/assets/15097156/bce5f1d7-fddb-404f-820e-a23a2dcdac0a)

Production also includes `Text` as part of the link.

Reply:
This might be a situation where we want to align the visual display of numbers based on what `va-telephone` is doing rather than try and duplicate what the existing VA.gov modal is doing.

So `<va-telephone sms` and `<va-telephone tty` are the desired visual displays even though they are different from what VA.gov has hard coded? 

With that said (and looking at [va-telelphone examples](https://design.va.gov/storybook/?path=/docs/components-va-telephone--default)), it seems like none of the numbers should be bold. We might want a designer to confirm though.

Reply:
Yeah that's a good point, we can ask in standup about this.